src/pages/MetricDetail.svelte

```diff
@@ -197,6 +225,15 @@
         {/each}
       </td>
     </tr>
+    {#if glamUrl}
+      <tr>
+        <td>GLAM</td>
```

Comment:
Would you prefer this titled "GLAM URL" instead?

Reply:
Nope! I think just `GLAM` is fine. We can add a help icon when @Fenn-CS finishes #293